test/store_unittest.cc

```diff
@@ -10,6 +10,10 @@ class MetadataStoreTest : public ::testing::Test {
  protected:
   MetadataStoreTest() : config(), store(config) {}
 
+  std::map<MonitoredResource, MetadataStore::Metadata> GetMetadataMap() const {
```

Comment:
Why is this needed? Isn't `GetMetadataMap` public?

Reply:
Backed out these changes - sorry about that, I should have remembered before pushing.